addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -643,7 +644,54 @@ registry.share = publicWidget.Widget.extend({
     },
 });
 
-registry.mediaVideo = publicWidget.Widget.extend({
+const MobileYoutubeAutoplayMixin = {
+    /**
+     * Takes care of any necessary setup for autoplaying video. In practice,
+     * this method will load the youtube iframe API for mobile environments
+     * because mobile environments don't support the youtube autoplay param
+     * passed in the url.
+     * @private
```

Comment:
Detail but we need a blank line between decription and `@private`, I can change it myself if you want, just remember for next times :+1:

Reply:
Not a problem, I updated it :+1: